### PR TITLE
Style updates

### PR DESCRIPTION
**Pull request by MichielLeyman**
- Increased gender color saturation for black bg readability
- Header is now overlaying map, not a separate element
- EqualStreetNames.Brussels name
- Increased breatheability of popup boxes, but reduced vertical spacing between wikidata and street names
- Changed overlays to darker background and adjusted font colors
- Changed font of entity names in popup boxes
- Added slight popup fadein animation
- Added low highlight in popup based on wikidata gender
- Add a desaturated color for the female highlight to keep it purple

### CODE REVIEW

File: public/en/index.html [hunk 1]
```diff
@@ -66,7 +66,7 @@
                     <li class="gender-chart-label" data-gender="f">
                         <span class="fa-li"><i class="fas fa-lg fa-venus"></i></span>
                         Female:
-                        <span class="gender-chart-count"></span>
+                        <span class="gender-chart-count"></span></span>
```
Comment (jbelien) [inline public/en/index.html]: There is a `</span>` too much !
```suggestion
                        <span class="gender-chart-count"></span>
```

File: resources/popup.ts [hunk 2]
```diff
@@ -1,24 +1,27 @@
...
   if (
     typeof wikidata !== "undefined" &&
     wikidata !== null &&
     // to-do: Find out why wikidata is a "null" string (instead of null)
     wikidata !== "null"
   ) {
+    const highlightColor = gender ? colors[gender.toLowerCase()] : "#fff";
     html += '<div class="popup-wikidata">';
-    html += `<div class="popup-name">${name}</div>`;
+    html += `<h2 class="popup-name highlight-low" style="background:linear-gradient(180deg, rgba(255, 255, 255, 0) 65%, ${highlightColor} 65%);">${name}</h2>`;
```
Comment (jbelien) [inline resources/popup.ts]: To be honest, I don't really like this highlight.
Good idea to "highlight" the gender but let's find another way !

File: resources/style.css [hunk 3]
```diff
@@ -7,28 +7,59 @@ body {
   height: 100vh;
 }
 
+nav.navbar {
+  background-color: rgba(52, 51, 50, 0.8);
+  position: fixed;
```
Comment (jbelien) [inline resources/style.css]: Unnecessary since there is the `fixed-top` class
```suggestion
```

File: resources/style.css [hunk 4]
```diff
@@ -7,28 +7,59 @@ body {
   height: 100vh;
 }
 
+nav.navbar {
+  background-color: rgba(52, 51, 50, 0.8);
```
Comment (jbelien) [inline resources/style.css]: I would increase the opacity to `0.9` to improve readability of the menu on mobile.

```suggestion
  background-color: rgba(52, 51, 50, 0.9);
```

File: resources/style.css (hunk 3 above)
Comment (MichielLeyman) [reply resources/style.css]: Thanks, forgot to remove after I found out that was a thing

File: resources/popup.ts (hunk 2 above)
Comment (MichielLeyman) [reply resources/popup.ts]: Do you dislike the style or the way it is applied?

**Pull request merged**